benchmark/infrastructure/BenchmarkMain.cpp

```diff
@@ -158,6 +187,13 @@ int main(int argc, char** argv) {
   po::store(po::parse_command_line(argc, argv, options), vm);
   po::notify(vm);
 
+  // Apply GROUP BY sampling params to runtime parameters
+  RuntimeParameters().set<"group-by-sample-enabled">(sampleEnabled);
+  RuntimeParameters().set<"group-by-sample-max-rows">(maxSampleRows);
+  RuntimeParameters().set<"group-by-sample-distinct-ratio">(
+      sampleDistinctRatio);
+  RuntimeParameters().set<"group-by-sample-group-threshold">(groupThreshold);
+
```

Comment:
As this file is a generic main function for all benchmarks, 
It shouldn't be modified, but all specific changes should be in the specific benchmark.
Also: Please move all the benchmarking stuff into a separate branch/PR s.t. we can focus on the sampling functionality in this branch.